db/dcrpg/internal/blockstmts.go

```diff
@@ -99,6 +99,15 @@ const (
 
 	SelectBlockChainRowIDByHash = `select block_db_id from block_chain where this_hash = $1;`
 
+	// SelectBlocksTicketsPrice selects the ticket price and difficulty for the 144th block
+	SelectBlocksTicketsPrice = `select sbits, time, difficulty from blocks where height % 144 = 0 order by time;`
+
+	SelectBlocksBlockSize = `Select time, size, numtx, height from blocks order by time;`
+
+	IndexBlocksTableOnHeight = `CREATE INDEX uix_height_index ON blocks(height);`
```

Comment:
Can we call this `uix_block_height` so that it's clear what table it's on from the name, and the `_index` is implied by the `uix` part (unique index)... well, it's not a UNIQUE, but that's fine.

Reply:
Also, please move all the statements on the `blocks` table up to the others.  This area of the file deals with the `block_chain` table.

Reply:
ok. Noted